collection-views-tests.js

```diff
@@ -258,6 +258,99 @@ Tinytest.add('where - upsert - t4 (no matches found, insert still works)', funct
   test.equal(typeof result.insertedId, 'string');
 });
 
+// Test if field specifiers work properly
+Tinytest.add('where - field specifiers - t1 (one specifier: only fetch the title)', function (test) {
+  var Books = newCollection();
+  var doc = Books
+    .where({ genre: 'science' }, { title: 1 })
+    .findOne({ kind: 'magazine' });
+  test.equal(doc.title, 'Science Magazine 1');
+  test.isNotUndefined(doc.title);
+  test.isUndefined(doc.genre);
+  test.isUndefined(doc.kind);
+  test.isUndefined(doc.catalogId);
+  test.isNotUndefined(doc._id);
+  test.isNotNull(doc._id);
+  test.equal(typeof doc._id, 'string');
+});
+
+Tinytest.add('where - field specifiers - t2 (two chained specifiers: only fetch the title and the kind)', function (test) {
+  var Books = newCollection();
+  var doc = Books
+    .where({}, { title: 1 })
+    .where({ genre: 'science' }, { kind: 1 })
```

Comment:
I'm not sure if I understand correctly, but here we are testing if we can use more than one field specifier in separate chained where statements. That's why there are 2 fields instead of just the 1 from above.